app/src/main/java/com/sanjeevyadavit/magecart/data/remote/dto/StoreConfigsDto.kt

```diff
@@ -1,9 +1,12 @@
-package com.sanjeevyadavit.magecart.model
+package com.sanjeevyadavit.magecart.data.remote.dto
 
 
 import com.google.gson.annotations.SerializedName
+import com.sanjeevyadavit.magecart.domain.model.StoreConfigs
 
-data class StoreConfigs(
+
+// QUESTION: Does DTO and creating corresponding model class in domain make sense or should we use single class?
```

Comment:
Yes ideally we should have both these.. your api contract can give some data in the response for which you would want to have different domain classes as per your business logic. So its better to have these two separate.